_posts/2017-1-20-typing-the-debugger.markdown

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title:  "How the Debugger got into the flow"
+date:   2017-1-20
+categories: DevTools JS
+---
+
+Last spring we kicked off a re-write of the Debugger frontend with the goal of being as developer friendly as possible.  
```

Comment:
I link to this article when referencing the re-write https://medium.com/mozilla-tech/introducing-debugger-html-8629b943dcf6#.kwp9uxevn